generateQuestions.js

```diff
@@ -0,0 +1,15 @@
+const handleObject = require('./handleObject');
```

Comment:
I'm not sure if this is supported with a CLI implementation, but this could potentially be refactored to "import { handleObject } from './handleObject' (?)

Reply:
Depends on the version of Node you're running but yes it was recently included as an experimental feature in Node 12. https://thecodebarbarian.com/nodejs-12-imports